recipes/max-cache-entries/index.html

```diff
@@ -0,0 +1,56 @@
+<html>
+  <head>
+    <title>Max Cache Entries Demo</title>
+    <link rel="stylesheet" href="../common.css">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+
+  <body>
+    <h1>Max Cache Entries Demo</h1>
+
+    <h3>Background</h3>
+    <p>
+      This configuration in
```

Comment:
Wordsmithing:

*This example demonstrates using the `maxCacheEntries` option to restrict the number of items that will be stored in the cache. It uses a dedicated cache to hold YouTube video thumbnails. That dedicated cache will...* [continue as before]